ethcore/src/pod_state.rs

```diff
@@ -26,7 +26,7 @@ use types::state_diff::StateDiff;
 use ethjson;
 
 /// State of all accounts in the system expressed in Plain Old Data.
-#[derive(Debug, Clone, PartialEq, Eq, Default)]
+#[derive(Debug, Clone, PartialEq, Eq, Default, Serialize)]
 pub struct PodState (BTreeMap<Address, PodAccount>);
```

Comment:
Not related to this PR, but there's an extra space between `PodState` and `(BTreeMap`.